docs/conceptual/iommu.rst

```diff
@@ -0,0 +1,98 @@
+.. meta::
+   :description: Input-Output Memory Management Unit (IOMMU)
+   :keywords: IOMMU, DMA, PCIe, XGMI, AMD, ROCm
+
+****************************************************************
+Input-Output Memory Management Unit (IOMMU)
+****************************************************************
+
+Overview
+================================================================
+
+The IOMMU (IO Memory Management Unit) is a device which provides memory remapping services to 
+IO devices on a platform. Similar to the MMU on a CPU, the IOMMU is able to provide a private 
+virtual address space for each device or group of devices on a platform. While the IOMMU is 
```

Comment:
By "platform", do we mean operating systems, or an entire OS-GPU config?

Reply:
IOMMU generally provides IO remapping services for all devices in a system (aka platform).

Reply:
That's not really answering my question.

When the term "platform" is being used here, what is it referring to? I need specifics so I can be as clear as possible in the text.

Reply:
platform = the system.  The motherboard and CPU and all of the devices connected to it.  If you take a server with a Zen CPU in it, the IOMMU integrated into the CPU provides IO and interrupt remapping services to all of the devices on the system (PCIe devices, USB, SATA, M.2, etc.).